gate-integrations-gremlin/src/main/groovy/com/netflix/spinnaker/gate/controllers/gremlin/GremlinController.groovy

```diff
@@ -0,0 +1,35 @@
+package com.netflix.spinnaker.gate.controllers.gremlin
+
+import com.netflix.spinnaker.gate.services.gremlin.GremlinService
+import io.swagger.annotations.ApiOperation
+import org.springframework.beans.factory.annotation.Autowired
+import org.springframework.boot.autoconfigure.condition.ConditionalOnExpression
+import org.springframework.web.bind.annotation.RequestBody
+import org.springframework.web.bind.annotation.RequestMapping
+import org.springframework.web.bind.annotation.RequestMethod
+import org.springframework.web.bind.annotation.RestController
+
+@RestController
+@RequestMapping('/gremlin')
+@ConditionalOnExpression('${integrations.gremlin.enabled:true}')
```

Comment:
this is new, and I've tested it.  only other change in this file is the package (addition of `gremlin`)

Reply:
Won't this default it to enabled (whereas the bean will default to disabled)?

Reply:
When I tested this locally, setting the value of the property in `gate.yml` to `false` did not inject this `RequestMapping`.  When the UI made the call, it 404ed, as expected.  

When I set it to `true`, the endpoint was properly mapped and made the appropriate redirection to the Gremlin API.

Reply:
I'm referring to the case where the config setting is absent entirely---if you don't have the `integrations.gremlin.enabled` key in your environment at all.  Then:
* the bean checks `@ConditionalOnProperty('integrations.gremlin.enabled')` and does not get created
* the controller checks `@ConditionalOnExpression('${integrations.gremlin.enabled:true}')`, falls back to the default of `true` in the expression, and gets created.

Then the controller tries to autowire the bean, and gate fails to start up.

Reply:
I misunderstood the syntax of the annotation - thanks for the walkthrough!  Changed as you suggested and manually tested all three variants: `true` / `false` / missing.